docs/SMCWG-charter.md

```diff
@@ -7,9 +7,9 @@ assist the interpretation of this Charter.
 
 An S/MIME certificate contains a public key bound to an email address; and may
 also contain the identity of a natural person or legal entity that controls such
-email address. The key pair can then be used to sign, encrypt, and decrypt
-email. An S/MIME certificate can be identified by the existence of an Extended
-Key Usage (EKU) Object Identifier (OID) of `1.3.6.1.5.5.7.3.4` for
+email address. The key pair can then be used to sign, verify, encrypt, and 
```

Comment:
From https://github.com/sleevi/cabforum-docs/pull/9/files/7d20e6bcfb580b409a37b8f2f1d0b43d6d10e749:

Key pairs can also be used to verify emails.